temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflowWithCronScheduleImpl.java

```diff
@@ -34,6 +36,8 @@
 public class TestWorkflowWithCronScheduleImpl implements TestWorkflowWithCronSchedule {
 
   public static final Map<String, AtomicInteger> retryCount = new ConcurrentHashMap<>();
+  public static final Hashtable<String, HashMap<Integer, String>> lastCompletionResults =
```

Comment:
Please don't use Hashtables, it's effectively deprecated. See ConcurrentHashMap above.
Also, please convert it into Map<String, Map<Integer, String>>. I'm not sure why the second generic is HashMap here. It's always better to use the most generic type.